libs/platform/src/lib/components/menu-button/menu-button.component.html

```diff
@@ -8,8 +8,10 @@
     [disabled]="disabled"
     (click)="onButtonClick($event)"
     [fdType]="type"
+    tabindex="0"
+    [attr.title]="title"
+    [attr.aria-haspopup]="true"
+    [attr.aria-expanded]="false"
```

Comment:
You can remove `aria-haspopup` and `aria-expanded`. The `menuTriggerFor` directive adds these attributes automatically.

Reply:
removed aria-haspopup and aria-expanded.